docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+
+### Steps to Dynamically Provision a Block Volume on Vanilla Kubernetes Cluster
+
+  This section describes the step by step instructions to provision a PersistentVolume dynamically on a `Vanilla Kubernetes` Cluster
+
+* Define a Storage Class as shown [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/example/example-sc.yaml)
+
+* Import this `StorageClass` into `Vanilla Kubernetes` Cluster:
+
+```text
+    kubectl create -f example-sc.yaml
+```
+
+* Define a `PersistentVolumeClaim` request as shown in the spec [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/example/example-pvc.yaml)
+
+* Import this `PersistentVolumeClaim` into `Vanilla Kubernetes` Cluster:
+
+```text
+    kubectl create -f example-pvc.yaml
+```
+
+* Verify the PersistentVolumeClaim was created:
+
+  Check to see if the `PersistentVolumeClaim` we just imported was created and has a `PersistentVolume` attached to it.
+
+  The `Status` section below should show `Bound` if it worked and the `Volume` field should be populated.
+
+  A `PersistentVolume` is automatically created and is bound to this `PersistentVolumeClaim`.
+
+```text
+  $ kubectl describe pvc example-vanilla-block-pvc
+    Name:          example-vanilla-block-pvc
+    Namespace:     default
+    StorageClass:  example-vanilla-block-sc
+    Status:        Bound
+    Volume:        pvc-7ed39d8e-7896-11ea-a119-005056983fec
+    Labels:        <none>
+    Annotations:   pv.kubernetes.io/bind-completed: yes
+                   pv.kubernetes.io/bound-by-controller: yes
+                   volume.beta.kubernetes.io/storage-provisioner: csi.vsphere.vmware.com
+    Finalizers:    [kubernetes.io/pvc-protection]
+    Capacity:      5Gi
+    Access Modes:  RWO
+    VolumeMode:    Filesystem
+    Mounted By:    <none>
+    Events:
+      Type    Reason                 Age                From                                                                                                 Message
+      ----    ------                 ----               ----                                                                                                 -------
+      Normal  Provisioning           50s                csi.vsphere.vmware.com_vsphere-csi-controller-7777666589-jpnqh_798e6967-2ce1-486f-9c21-43d9dea709ae  External provisioner is provisioning volume for claim "default/example-vanilla-block-pvc"
+      Normal  ExternalProvisioning   20s (x3 over 50s)  persistentvolume-controller                                                                          waiting for a volume to be created, either by external provisioner "csi.vsphere.vmware.com" or manually created by system administrator
+      Normal  ProvisioningSucceeded  8s                 csi.vsphere.vmware.com_vsphere-csi-controller-7777666589-jpnqh_798e6967-2ce1-486f-9c21-43d9dea709ae  Successfully provisioned volume pvc-7ed39d8e-7896-11ea-a119-005056983fec
+```
+
+* Verify a PersistentVolume was created
```

Comment:
`PersistentVolume`

Reply:
I didn't understand, can you please explain your comment?

Reply:
I mean to add ` to wrap around PersistentVolume like what you did on line 90.